packages/evolution-generator/src/scripts/labels_generator.py

```
@@ -157,14 +157,16 @@ def delete_all_labels_yaml_files(
                         raise e
 
     @classmethod
-    def add_translation(cls, language, section, path, value, rowNumber, translations):
+    def add_translation(
```

Ruff (0.15.12)

[warning] Missing return type annotation for classmethod `add_translation`

Add return type annotation: `None`

(ANN206)